### ISSUE TITLE
Equivalent of pandas.DataFrame.describe

### ISSUE THREAD

**Issue by amirouche** [reubano/meza#15]
Is there an equivalent of [pandas.DataFrame.describe](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.describe.html).

TIA!

**Comment by amirouche** [comment]
It's seems like there none. Can some scratch an idiomatic way to write something like that in `meza`

**Comment by reubano** [comment]
Correct, there is none. While something like this is possible, care must be taken to ensure not too much memory is used up in the case of large record iterators.
